integration/client/tracing_test.go

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
Most of the integration test for image pull are in [client_test.go](https://github.com/containerd/containerd/blob/4bd8b5be444a8c381db7fa678def3bb6d416d0e1/integration/client/client_test.go). WDYT about adding this test to that file itself?